packages/mobile/src/analytics/Properties.tsx

```diff
@@ -1114,6 +1117,16 @@ interface DappKitProperties {
   }
 }
 
+interface CICOEventsProperties {
+  [CICOEvents.link_bank_account_cancel]: undefined
```

Comment:
Why are these all `undefined`?

Reply:
@jh2oman I was following the pattern used by other events which don't require additional properties to be sent; I'm not totally sure if this is object is needed, but felt it would be good to be consistent with our other events. I'm happy to remove this if it's totally unnecessary.

Reply:
Ah, don't require additional properties. I think its fine to continue the pattern. Its nice and explicit then.